test/cases/040_packages/005_extend/002_xfs/check.sh

```diff
@@ -9,5 +9,5 @@ function failed {
 
 [ -f /var/lib/docker/bar ] || failed
 touch /var/lib/docker/foo || failed
-df -h | grep -q "507.7M" || failed
+df -h | grep -q "504.8M" || failed
```

Comment:
I would be reluctant to check that exactly - check the first digit or something, a different version may have slightly different space usage.